x-pack/plugins/observability_solution/apm/public/components/app/service_map/cytoscape_options.ts

```diff
@@ -79,25 +79,30 @@ function getBorderWidth(el: cytoscape.NodeSingular) {
 // @ts-expect-error `documentMode` is not recognized as a valid property of `document`.
 const isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
 
-export const getAnimationOptions = (theme: EuiTheme): cytoscape.AnimationOptions => ({
-  duration: parseInt(theme.eui.euiAnimSpeedNormal, 10),
+export const getAnimationOptions = (euiTheme: EuiThemeComputed): cytoscape.AnimationOptions => ({
+  // @ts-expect-error
+  duration: parseInt(euiTheme.animation.normal, 10),
   // @ts-expect-error The cubic-bezier options here are not recognized by the cytoscape types
-  easing: theme.eui.euiAnimSlightBounce,
+  easing: euiTheme.animation.bouncetheme,
```

Comment:
`bouncetheme` does not exist on `euiTheme` 🤔 
Should this be `euiTheme.animation.bounce`?

Reply:
the dangers of using ` @ts-expect-error`

Reply:
typo, fixed